tests/src/FunctionalJavascript/CorporateContactFormTest.php

```diff
@@ -517,6 +532,54 @@ protected function assertTopicAjax($delta = 0): void {
     $this->assertNotEmpty($element);
   }
 
+  /**
+   * Asserts language options fields.
+   *
+   * @param string $field_name
+   *   Field name.
+   */
+  protected function assertLanguageOptions(string $field_name): void {
+    /** @var \Behat\Mink\WebAssert $assert */
+    $assert = $this->assertSession();
+    /** @var \Behat\Mink\Element\DocumentElement $page */
+    $page = $this->getSession()->getPage();
+    $field_name_selector = str_replace('_', '-', $field_name);
+
+    // Assert there are 25 input elements with 24 languages in specific order
+    // and last empty element.
+    $input_elements = $page->findAll('css', '[data-drupal-selector="edit-corporate-fields-override-languages-' . $field_name_selector . '"] .form-autocomplete');
+    $this->assertEquals(25, count($input_elements));
+    $expected_values = [
+      0 => 'Bulgarian (http://publications.europa.eu/resource/authority/language/BUL)',
+      1 => 'Spanish (http://publications.europa.eu/resource/authority/language/SPA)',
+      2 => 'Czech (http://publications.europa.eu/resource/authority/language/CES)',
+      3 => 'Danish (http://publications.europa.eu/resource/authority/language/DAN)',
+      21 => 'Slovenian (http://publications.europa.eu/resource/authority/language/SLV)',
+      22 => 'Finnish (http://publications.europa.eu/resource/authority/language/FIN)',
+      23 => 'Swedish (http://publications.europa.eu/resource/authority/language/SWE)',
+    ];
```

Comment:
Either we assert all or nothing, I don't see the point of this random assertion.

Reply:
Such approach is already used here in https://github.com/openeuropa/oe_contact_forms/blob/e8058ad393eb4ccaac4c7fa736911658e98142e4/tests/src/FunctionalJavascript/MessageFormTest.php#L102.

In case of 1000 items should we set all of them? IMHO such checks validates proper order.

Reply:
Yes, but here it's 24 items. Anyway, mine was just a suggestion.

Reply:
Here 24 items, and below in 2 places 24 items (72 of total) - this test will be just a list of expected options. So let's keep it as is to make the test readable.